Button/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import MaterialComponent from '../MaterialComponent';
+import MaterialComponent from '../ts/MaterialComponent';
```

Comment:
I thought all `.d.ts` should now be a build artifact, hence should not be in the git repo at all. isn't it?

Reply:
I'm not there yet going Z-A

Reply:
Also the /_Component_/index.d.ts are manual written, but are just reexports that havn't to rewritten ever.
Template:
```
import _Component_ from '../ts/_Component_';

export default _Component_;
export * from '../ts/_Component_';
```

Reply:
i thought this will be auto generated, because we'll need the same in esm folder.
can this be auto generated?

I see that the final step will only have folder `/ts` in git repo and will have `/` and `/ -> /esm` folder in npm

Reply:
Yes I will look into that after the tests are passing again, it's just copying them from `/esm`
And I would also include the `/ts` folder, for those who want to use typescript and want it raw (like me)

Reply:
Now auto-generated